Cleanup erroneous relationships

tooomm opened the pull request:
It was not just Kiora that needed a fix. ~~There are probably still more.~~

This also removes double linkings from both directions for some tokens via `related` and `reverse-related`.

<br>

Linking https://github.com/Cockatrice/Magic-Token/commit/0aa71e3d538ce355fe1b1d7ca9539a3510d08e7e

Psithief commented:
I think perhaps there's more than one change here so please give me a bit to examine this.

tooomm commented:
>I think perhaps there's more than one change here so please give me a bit to examine this.

Absolutely, please always double check! 👍 

<br>

If you follow the commits one by one it should be straight forward:

1) Removed two erroneous walker relation (+fixed a token count along the way)
2) Cleaned up the now present double linking (removes the fixed token count from 1) completely)
3) Removed another erroneous walker relation

Psithief commented on tokens.xml:
```diff
@@ -8460,7 +8459,7 @@ Sacrifice this creature: This creature deals 1 damage to target creature or play
             <reverse-related>Liliana, Dreadhorde General</reverse-related>
             <reverse-related>Liliana, Heretical Healer</reverse-related>
-            <reverse-related count="x">Liliana, the Last Hope</reverse-related>
+            <reverse-related count="x">Liliana, the Last Hope (Emblem)</reverse-related>
```
The minimum number of zombies is 2, so what about `"x=2"` here?

tooomm replied:
Sure, that works fine.

The x count and it's "correct usage" in various situations is still a bit obscure.
No consensus in the file or the people working with it. That makes it a bit tricky to find the "right" solution as there are often several ways to utilize it.

Psithief commented:
The weird part is, I think I kept doing it because I thought the first time was on purpose. 
This will be good to clean up.
